DigitalLearningSolutions.Web/Views/Home/LearningContent.cshtml

```diff
@@ -0,0 +1,75 @@
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.Home
+@model LandingPageViewModel
+
+@{
+  ViewData["Title"] = "Welcome - Learning Content";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/learningContent.css")"/>
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">

```

Comment:
Out of curiosity, what are these ids used for?

Reply:
Not sure. That was copied from the welcome page I was working from. Want me to check?

Reply:
Yes please, it doesn't look like they're used for styling and we don't need the ids in code.